packages/analytics-plugin-google-analytics/src/node.js

```diff
@@ -7,12 +7,24 @@ if (!process.browser) {
 /**
  * Serverside Google Analytics plugin
  * @param {object} pluginConfig - Plugin settings
- * @param {string} pluginConfig.trackingId - Google Analytics site tracking Id
+ * @param {string} pluginConfig.trackingId - Google Analytics site tracking Id. Same as tid (se below), only one needed
+ * @param {string} pluginConfig.tid - Google Analytics site tracking Id. Same as trackingId (se above), only one needed
```

Comment:
The `tid` property looks like a repeat of `trackingId` do we need that or can we just use the `trackingId` field?

Reply:
Yes, it is indeed. 
I added this 4 new docummented attributes of `pluginConfig` but there are more that can be used in `universal-analytics` [initialization](https://github.com/peaksandpies/universal-analytics/blob/master/lib/index.js#L19).
I keeped the `trackingId` attribute in order to keep compatibility with the actual implementation and also allow to define all the attributes accepted in `universal-analytics` initialization.
Maybe it would be better to keep the actual jsDoc and add only a reference to all the possible attributes that can be used in `pluginConfig` and explaining that the `trackingId` attribute can be declared also as `tid`.